builtin/mainmenu/tab_settings.lua

```diff
@@ -333,6 +359,52 @@ local function handle_settings_buttons(this, fields, tabname, tabdata)
 		ddhandled = true
 	end
 
+	for i = 1, #labels.shadow_levels do
+		if fields["dd_shadows"] == labels.shadow_levels[i] then
+			core.settings:set("shadow_levels", dd_options.shadow_levels[2][i])
+			ddhandled = true
+		end
+	end
+
+	if fields["dd_shadows"] == labels.shadow_levels[1] then
+		core.settings:set("enable_dynamic_shadows", "false")
+	else
+		core.settings:set("enable_dynamic_shadows", "true")
+		core.settings:set("shadow_strength", "0.35")
```

Comment:
If this is the suggested value shouldn't it just be made default?

Reply:
It's the default value, yes. I don't if I must put it in another place, it's already in defaultsettings.cpp

Reply:
defaultsettings has `"shadow_strength", "0.25"` which is the reason I'm asking.